wine_cellar/react/maps/Map.jsx

```diff
@@ -0,0 +1,33 @@
+import React, { useImperativeHandle } from 'react'
+import { MapContainer, useMap } from 'react-leaflet'
+import MaplibreGlLayer from './MaplibreGlLayer'
+
+const Map = React.forwardRef(function Map(
+  { attribution, baseUrl, polygon, omtToken, children, ...rest },
+  ref
+) {
+  const MapLayers = () => {
+    const map = useMap()
+    useImperativeHandle(ref, () => map)
+    return (
+      <>
+        <MaplibreGlLayer attribution={attribution} baseUrl={baseUrl} />
+      </>
+    )
+  }
+
+  return (
+    <MapContainer
+      style={{ minHeight: "60vh" }}
```

Comment:
I've set it to 60vh here but not sure if there's a better solution

Reply:
@goapunk did you see this thread? I can have a play with it but just don't want to double on attempts :) https://stackoverflow.com/questions/41660648/make-react-leaflet-map-component-resize-itself-to-fit-available-space

Reply:
@goapunk will take a look at this now if you're still not on it?